catalogs/data/bedrock/terraform/aws/context.tf

```diff
@@ -0,0 +1,8 @@
+data "plural_service_context" "mgmt" {
+  name = "plrl/clusters/${var.cluster}"
+}
+
+locals {
+  configuration    = jsondecode(data.plural_service_context.mgmt.configuration)
+  eks_cluster_name = lookup(local.configuration, "cluster_name", "default-cluster")
```

Comment:
If you didn't add the default here, would this just throw an exception if not found?  That'd actually be preferable behavior I think